src/applications/disability-benefits/2346/schemas/address-schema.js

```diff
@@ -361,6 +361,15 @@ export const addressUISchema = (
         hideIf: () => !isMilitaryBaseAddress,
         hideOnReviewIfFalse: true,
         useDlWrap: true,
+        updateSchema: (formData, schema, uiSchema) => {
+          const countryFormData = get(path, formData);
+          if (countryFormData.country === 'ARMED FORCES AF,EU,ME,CA') {
```

Comment:
So we have access to the schema and what we would normally expect a country code to be.. can we do something like 
```
if (expectedSchema.includes(countryFormData.country) {
  return false
}
```

I don't think the specificity you put in to this conditional is appropriate.  If any casing is different or if one letter is different - this condition will not be met and the bug will still exist

Reply:
expectedSchema is just some pseudo code for an array of what our input field is expecting